tests/unit/customizations/cloudformation/test_deployer.py

```diff
@@ -117,7 +118,7 @@ def test_create_changeset_success(self):
             "Capabilities": capabilities,
             "Description": botocore.stub.ANY,
             "RoleARN": role_arn,
-            "NotificationARNs": notification_arns
+            "NotificationARNs": notification_arns,
```

Comment:
These parameters don't accept an s3_uploader - these are used in the stubed call to boto3 create_change_set, which accepts a TemplateBody or a TemplateURL. A test has been added for deploying the change set using TemplateURL rather than TemplateBody. 

We now use a TemplateURL if an s3_uploader is passed, otherwise TemplateBody is used. By default, the s3_uploader is None, so we use TemplateBody by default (as it was before).